internal/compiler/widgets/native/std-widgets.slint

```diff
@@ -135,13 +135,31 @@ export ComboBox := NativeComboBox {
 export TabWidgetImpl := NativeTabWidget { }
 export TabImpl := NativeTab { }
 
-export TabBarImpl := HorizontalLayout {
+export TabBarImpl := Rectangle {
     // injected properties:
     property<int> current; // The currently selected tab
-    property<int> current-focused; // The current focused index (or -1 if no focus)
+    property<int> current-focused: current; // The currently focused tab
     property<int> num-tabs; // The total number of tabs
 
-    alignment: start;
+    HorizontalLayout {
+        spacing: 8px;
+        alignment: start;
+        @children
+    }
+
+    fs := FocusScope {
+        key-pressed(event) => {
```

Comment:
No handling for activation?

Apart from this, is there any difference between this TabBarImpl and the one in the fluent style?

Should the 8px spacing come from the native style metrics?

Reply:
> Apart from this, is there any difference between this TabBarImpl and the one in the fluent style?

It seems not.  So this could be moved in the common/common.slint and re-exported

Reply:
But the spacing is indeed suspicious.